task/oci-copy/0.1/README.md

```diff
@@ -14,6 +14,7 @@ Note: the bearer token secret, if specified, will be sent to **all servers liste
 |IMAGE|Reference of the image buildah will produce.||true|
 |OCI_COPY_FILE|Path to the oci copy file.|./oci-copy.yaml|false|
 |BEARER_TOKEN_SECRET_NAME|Name of a secret which will be made available to the build as an Authorization header. Note, the token will be sent to all servers found in the oci-copy.yaml file. If you do not wish to send the token to all servers, different taskruns and therefore different oci artifacts must be used.|"does-not-exist"|false|
+|AWS_SECRET_NAME|Name of a secret which will be made available to the build to construct Authorization headers for requests to Amazon S3. If specified, this will take precedence over BEARER_TOKEN_SECRET_NAME.|does-not-exist|false|
```

Comment:
Hmm, so it's not possible to do both? I.e. all the authenticated parts must come _either_ from S3 _or_ from servers that use bearer auth?

I guess that's fine if it meets the current use cases. Wondering what would be the path to make this task more generically usable though